docs/src/administration/configuration.md

```diff
@@ -88,6 +88,14 @@ The following includes ThreadPool related parameters, which can be passed in thr
 - a PKCS#8 private key file in PEM format. e.g. /home/tidb/client-key.pem.
 - default: null
 
+#### tikv.tls.reload_interval
+- The interval in seconds to poll the change of TLS context, if a change detected, the SSL context will be rebuilded.
+- default: `"10s"`, `"0s"` means disable TLS context reload.
+
+#### tikv.conn.recycle_time
+- After `tikv.conn.recycle_time` (in seconds) with a TLS context reloading, the old connections will be forced to shutdown preventing channel leak.
```

Comment:
will be forced to shutdown after recycle time to prevent channel leak.

Reply:
Done.